nvflare/private/fed/server/job_runner.py

```diff
@@ -189,6 +189,7 @@ def run(self, fl_ctx: FLContext):
                     if ready_job:
```

Comment:
for this "run" function,
if the job_manager is None in L181,
this function will then be doing nothing, it is not getting new values of job_manager.